packages/ui/src/components/va-checkbox/VaCheckbox.vue

```diff
@@ -90,6 +91,7 @@ export default defineComponent({
     indeterminateIcon: { type: String as PropType<string>, default: 'remove' },
     id: { type: String as PropType<string>, default: '' },
     name: { type: String as PropType<string>, default: '' },
+    ariaLabel: { type: String, default: null },
```

Comment:
`default: null` is it on purpose?

Reply:
Thought it was, but after check have found out, that it's not.